### DISCUSSION TITLE
Press `.` on the "Code" page

### DISCUSSION THREAD

**Discussion by Steffo99** [ReactiveDrop/reactivedrop_translations#201] [Ideas]
<details>
<summary>
Click here if you don't want to discover it yourself
</summary>
<br>
It will take you directly to a browser instance of Visual Studio Code editing this repository, with the possibility of installing the recommended extensions and of creating pull requests directly.

![A screenshot of the github.dev editor](https://user-images.githubusercontent.com/1540885/181678821-47c539d5-0710-4b3d-9015-5adbcf410d0f.png)

[GitHub Documentation about the github.dev editor](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor)

</details>

I think it would be a good idea to add this as a note somewhere, since I think it would be very helpful to new (and possibly old) translators!

**Comment by Duke-M-commits** [comment]
Awesome, I totally forgot that the greatest editor of all time just casually runs in every browser.
Please add it yourself, more people need to add stuff to the wiki. Like "how to translate using your webbrowser" or something.
On a sidenote, I'm happy that it seems to be helpful :relaxed:

PS: Screenshot tools adding their own ui to the content ... ahhhh ... classic (:eyes: at you steam-overlay :angry:)

**Comment by Duke-M-commits** [comment]
There's now an even better way, just open a codespace. The whole repo gets loaded, with [recommended extensions](https://github.com/ReactiveDrop/reactivedrop_translations/wiki/How-to-translate-using-vscode) like DeepL and Gitlens.
![image](https://user-images.githubusercontent.com/69652463/204561990-956fa0d7-df0b-44b8-a7c7-fb2f4fd796a2.png)

**Reply by Steffo99** [reply]
Maybe I should mention Codespaces as well in the `CONTRIBUTING.md` file...
